CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+v2.6.9
+------
+
+* Deprecate Tasks.par(...) for safer alternative Task.par(...) that does not throw IllegalArgumentException on empty collection.
+* Enable automatic cross-thread stack tracing
```

Comment:
+ "It is optional feature, turned off by default. See ParSeqGlobalConfiguration. setCrossThreadStackTracesEnabled()."